packages/server/lib/privileged-commands/privileged-channel.js

```diff
@@ -95,7 +95,10 @@
       // stack URLs come in URI encoded by default.
       // we need to make sure our script names are also URI encoded
       // so the comparisons match
-      const scriptName = encodeURI(script)
+      let scriptName = encodeURI(script).replace(/%5C/g, '\\')
+
+      // we do NOT want to encode backslashes (\) as this causes pathing issues on Windows
+      scriptName = scriptName.replace(/%5C/g, '\\')
```

Comment:
Did you mean to call `replace(/%5C/g, '\\')` twice?

Reply:
thought I rebased that out. guess not 🥴 . It should only be once!